README.md

```diff
@@ -22,3 +22,18 @@ details about how to develop integrations with this SDK.
 
 An example template project can be found
 [here](https://github.com/JupiterOne/integration-sdk/tree/master/template).
+
+## Integration SDK Development
+
+If you are making changes to the integration SDK and you want to test the
+changes in another integration project then it is recommended to automatically
+rebuild and link this project when changes are made.
```

Comment:
I've wondered whether this workflow is supported by creating a Yarn workspace, that isn't checked in anywhere, but I've not messed with it enough to say for sure.

Reply:
I think it definitely can be supported via workspaces. But I suppose for others not using workspaces, the `autobuild` command could be helpful.